[DOCS] Update ILM screenshots and tutorial (#68482)
elastic/elasticsearch: docs/reference/ilm/example-index-lifecycle-policy.asciidoc

To complete this tutorial, you'll need:

* An {es} cluster with hot and warm data tiers.

** {ess}:
Elastic Stack deployments on the {ess} include a hot tier by default. To add a
warm tier, edit your deployment and click **Add capacity** for the warm data
tier.
+
[role="screenshot"]
image::images/ilm/tutorial-ilm-ess-add-warm-data-tier.png[Add a warm data tier to your deployment]

** Self-managed cluster:
Assign `data_hot` and `data_warm` roles to nodes as described in
<<data-tiers>>.
+
For example, include the `data_warm` node role in the `elasticsearch.yml` file
of each node in the warm tier:
+
[source,yaml]
----
node.roles: [ "data_warm" ]
----

* A host with the {agent} installed and configured to send logs to your {es}
cluster.

[discrete]
[[example-using-index-lifecycle-policy-view-ilm-policy]]
==== View the policy

The {agent} uses data streams with an index pattern of `logs-*-*` to store log
monitoring data. {es}'s built-in `logs` {ilm-init} policy automatically manages
backing indices for these data streams.

To view the `logs` policy in {kib}:

. Open the menu and go to **Stack Management > Index Lifecycle Policies**.
. Select the `logs` policy.

> The `logs` policy uses our recommended rollover defaults. These defaults
> initiate a rollover when the current write index reaches 50GB or becomes 30 days
> old.

Reviewer: ```suggestion
The `logs` policy uses the recommended rollover defaults: start writing to a new index when the current write index reaches 50GB or becomes 30 days old.
```

> To confirm these rollover settings, click **Settings** for the hot phase. Then

Reviewer: This seems like an odd step because of the UI--to view the settings, you have to disable the "use defaults" option, but we're telling them to confirm that they want to use the defaults. 

```suggestion
To view or change the rollover settings, click **Settings** for the hot phase. Then
```

disable **Use recommended defaults** to display the rollover settings.

[role="screenshot"]
image::images/ilm/tutorial-ilm-hotphaserollover-default.png[View rollover defaults]

[discrete]
[[ilm-ex-modify-policy]]
==== Modify the policy

> The `logs` policy's default settings are enough to prevent the creation of many
> tiny daily indices. You can modify the policy to meet more complex requirements.

Reviewer: ```suggestion
The default `logs` policy is designed to prevent the creation of many
tiny daily indices. You can modify the policy to meet your performance requirements and manage resource usage.
```

. Activate the warm phase and click **Settings**.
+
--
[role="screenshot"]
image::images/ilm/tutorial-ilm-modify-default-warm-phase-rollover.png[Add a warm phase with custom settings]

> .. Set *Move data into phase when* to *30 days*. This keeps indices in the hot
> tier for 30 days. After 30 days, indices move to the warm tier.

Reviewer: As mentioned above, this keeps the data in the hot phase for 30 days after rollover.
```suggestion
.. Set *Move data into phase when* to *30 days*. This moves indices to the warm tier 30 days after rollover.
```

.. Enable **Replicas** and change *Number of replicas* to *1*.

.. Enable *Force merge* and set *Number of segments* to *1*.
--

> . Activate the delete phase and set *Move data into phase when* to *90* days.

Reviewer: Might be worth clarifying here, too. "This moves indices to the delete phase 90 days after rollover."

**Number of segments**: 1

[role="screenshot"]
image::images/ilm/tutorial-ilm-custom-policy.png["Modify the custom policy to add a warm phase"]
--

. Activate the delete phase and set the timing to 90 days.
+
[role="screenshot"]
image::images/ilm/tutorial-ilm-delete-phase-creation.png["Delete phase"]

To configure the index to use the new policy:

. Go to Management and select *Index Lifecycle Policies*.
. Find your {ilm-init} policy and click its *Actions* link.
. Choose *Add policy to index template*.
. Select your {filebeat} index template name from the *Index template* list. For example, `filebeat-7.5.x`.
. Click *Add Policy* to save the changes.
+
NOTE: If you initially used the default {filebeat} {ilm-init} policy, you will
see a notice that the template already has a policy associated with it. Confirm
that you want to overwrite that configuration.

When you change the policy associated with the index template, the active
index will continue to use the policy it was associated with at index creation
unless you manually update it. The next new index will use the updated policy.
For more reasons that your {ilm-init} policy changes might be delayed, see

> <<update-lifecycle-policy, Update Lifecycle Policy>>.

Author: I removed this section entirely. I'm not really sure if there's an advantage to creating and applying a new policy. Updating the built-in index templates is also tricky.

Reviewer: Yeah, this doesn't look like something we want to encourage.
